### PR TITLE
Use v0.9.0 c/common/pkg/auth in login/logout

### PR DESCRIPTION
**Pull request by QiWang19**
- upgrade  c/common v0.9.0
- use shared code for login/logout from c/common v0.9.0
Signed-off-by: Qi Wang <qiwan@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/auth/login.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
+package auth
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	buildahcli "github.com/containers/buildah/pkg/cli"
```
Comment (rhatdan) [inline pkg/auth/login.go]: Is there a way we can do without this.  We don't want to suck buildah into this, if we are going to move it to common.

File: pkg/cli/common.go [hunk 2]
```diff
@@ -109,6 +109,19 @@ type FromAndBudResults struct {
 	Volumes      []string
 }
 
+// LoginReply represents the results for flags in login
```
Comment (rhatdan) [inline pkg/cli/common.go]: These should be defined in the new package not in here.

**Comment by rhatdan** [comment]
@QiWang19 Please rebase.

**Comment by rhatdan** [comment]
@QiWang19 Please rebase.
LGTM

**Comment by TomSweeneyRedHat** [comment]
Need to get the "WIP" off the title.  Otherwise LGTM.

**Comment by QiWang19** [comment]
> Need to get the "WIP" off the title. Otherwise LGTM.

Is this the reason I can't see CI running? I want to remove the "WIP" after https://github.com/containers/common/pull/106 got merged.

**Comment by TomSweeneyRedHat** [comment]
@QiWang19 IDK for sure, but when you're ready for it to be merged I'd remove the WIP from thetitle and repush.

**Comment by TomSweeneyRedHat** [comment]
@QiWang19 tests are running, but not happy now.  Looks like it can't find c/common/pkg/auth which I think is expected at the moment.

**Comment by QiWang19** [comment]
@TomSweeneyRedHat I should be able to edit go.mod to vendor my branch, but it is not vendored in...

**Comment by QiWang19** [comment]
@TomSweeneyRedHat  @rhatdan  PTAL
upgrade c/common v0.9.0
use shared code for login/logout from c/common v0.9.0

**Comment by rhatdan** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM, however I'd like to NOT merge this until Podman 1.9 finally goes out the door, especially since the Buildah 1.14.8 vendor was hiccuping.

**Comment by TomSweeneyRedHat** [comment]
bors retry

**Comment by TomSweeneyRedHat** [comment]
I luv bors at times
bors retry

**Comment by QiWang19** [comment]
bors retry

**Comment by TomSweeneyRedHat** [comment]
@QiWang19 can you try something gross for me please, but it might unstick bors.
Edit a file slightly in one of your files.
git add file
git commit --amend -s
Revert the edit
git add file
git commit --amend -s
fetch/rebase/push

If that doesn't work, we'll have to hunt down @cevich

**Comment by TomSweeneyRedHat** [comment]
Woot!  finally through!  Thanks for hanging in there for this one @QiWang19

**Comment by vrothberg** [comment]
Great work, @QiWang19 !

**Comment by rhatdan** [comment]
Now lets get it merged into skopeo.

**Pull request merged**